addons/auth_oauth/static/src/scss/auth_oauth.scss

```diff
@@ -1,18 +1,18 @@
     }
 
-    .fa-google-plus-square {
-        color: #de564a;
+    &.o_facebook_provider {
+        background-image: url("data:image/svg+xml,%3Csvg width='500' height='500' viewBox='0 0 500 500' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M500 250C500 111.93 388.07 0 250 0S0 111.93 0 250c0 117.24 80.72 215.62 189.61 242.64V326.4h-51.55V250h51.55v-32.92c0-85.09 38.51-124.53 122.05-124.53 15.84 0 43.17 3.11 54.35 6.21v69.25c-5.9-.62-16.15-.93-28.88-.93-40.99 0-56.83 15.53-56.83 55.9V250h81.66l-14.03 76.4H280.3v171.77C404.07 483.22 500 377.82 500 250Z' fill='%230866FF'/%3E%3Cpath d='m347.92 326.4 14.03-76.4h-81.66v-27.02c0-40.37 15.84-55.9 56.83-55.9 12.73 0 22.98.31 28.88.93V98.76c-11.18-3.11-38.51-6.21-54.35-6.21-83.54 0-122.05 39.44-122.05 124.53V250h-51.55v76.4h51.55v166.24c19.34 4.8 39.57 7.36 60.39 7.36 10.25 0 20.36-.63 30.29-1.83V326.4h67.64Z' fill='%23ffffff'/%3E%3C/svg%3E");
     }
 
-    .o_custom_icon {
-        margin: 0 0.15em;
-        width: 1em;
-        height: 1em;
-        border: 3px solid #714B67;
-        border-radius: 100%;
-        transform: translateY(2px);
+    &.o_google_provider {
+        background-image: url("data:image/svg+xml,%3Csvg width='32' height='32' viewBox='0 0 32 32' data-name='Layer 1' id='Layer_1' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M23.75,16A7.7446,7.7446,0,0,1,8.7177,18.6259L4.2849,22.1721A13.244,13.244,0,0,0,29.25,16' fill='%2300ac47'/%3E%3Cpath d='M23.75,16a7.7387,7.7387,0,0,1-3.2516,6.2987l4.3824,3.5059A13.2042,13.2042,0,0,0,29.25,16' fill='%234285f4'/%3E%3Cpath d='M8.25,16a7.698,7.698,0,0,1,.4677-2.6259L4.2849,9.8279a13.177,13.177,0,0,0,0,12.3442l4.4328-3.5462A7.698,7.698,0,0,1,8.25,16Z' fill='%23ffba00'/%3E%3Cpolygon fill='%232ab2db' points='8.718 13.374 8.718 13.374 8.718 13.374 8.718 13.374'/%3E%3Cpath d='M16,8.25a7.699,7.699,0,0,1,4.558,1.4958l4.06-3.7893A13.2152,13.2152,0,0,0,4.2849,9.8279l4.4328,3.5462A7.756,7.756,0,0,1,16,8.25Z' fill='%23ea4435'/%3E%3Cpolygon fill='%232ab2db' points='8.718 18.626 8.718 18.626 8.718 18.626 8.718 18.626'/%3E%3Cpath d='M29.25,15v1L27,19.5H16.5V14H28.25A1,1,0,0,1,29.25,15Z' fill='%234285f4'/%3E%3C/svg%3E");
```

Comment:
It's a good idea, it would be nice to have the real logo with colors indeed. But the result is quite pixelated on my screen:
![image](https://github.com/user-attachments/assets/0c0ffa40-e761-4164-ae2e-1d89d613131c)

It's a bit better with the icons increased in size a bit. Is it just my screen? Also, don't you think it would be even prettier if the icons were all aligned vertically? Something like that:
![image](https://github.com/user-attachments/assets/613a28bb-e0c2-41fe-b728-4337cc323f6c)